code/game/objects/items/legcuffs.dm

```diff
@@ -92,3 +93,49 @@
 					var/mob/living/simple_animal/SA = AM
 					SA.health -= 20
 	..()
+
+
+/obj/item/restraint/legcuffs/xeno_restraints
+	name = "xeno restraints"
+	desc = "Use this to hold xenomorphic creatures moderately-safely."
+	gender = PLURAL
```

Comment:
point of swapping to plural? should be neuter by default which in most cases I think is the same?